openwisp_controller/config/base/vpn.py

```diff
@@ -195,6 +213,13 @@ def save(self, *args, **kwargs):
         if not created:
             self._check_changes()
         create_dh = False
+        if self._is_backend_type('zerotier'):
+            config = deepcopy(self.config['zerotier'][0])
+            config['name'] = self.name
+            if created:
+                self._create_zerotier_server(config)
+            else:
+                self._update_zerotier_server(config)
```

Comment:
Yes, we can do that. However, I believe we can only move the `self._update_zerotier_server(config)` logic because when a new ZeroTier VPN server is created, we must perform the `save()` operation and set fields such as `network_id`. These fields are later used by `_update_zerotier_server()` to update VPN configurations.